saleor/order/actions.py

```diff
@@ -98,17 +97,14 @@ def order_confirmed(
     manager = get_plugins_manager()
     manager.order_confirmed(order)
     if send_confirmation_email:
-        send_order_confirmed.delay(order.pk, user.pk)
+        send_order_confirmed(order, user, manager)
 
 
 def handle_fully_paid_order(order: "Order", user: Optional["User"] = None):
     events.order_fully_paid_event(order=order, user=user)
-
+    manager = get_plugins_manager()
```

Comment:
you call this function in `order_captured` and there are already fetched manager, maybe you could pass it by argument  instead of calling it twice?